.github/workflows/build.yml

```
@@ -0,0 +1,129 @@
+name: Build and upload to PyPI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build_wheels:
+    name: Build ${{ matrix.build }} wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-20.04
+            build: "cp39-*x86_64*"
+          - os: ubuntu-20.04
+            build: "cp39-*i686*"
+          - os: windows-2019
+            build: "cp39-*win_amd64*"
+          - os: windows-2019
+            build: "cp39-*win32*"
+          - os: macos-10.15
+            build: "cp39-*x86_64*"
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # required for versioneer to find tags
+
+      - name: Install build dependencies
+        if: runner.os == 'Windows' || runner.os == 'macOS'
+        run: |
+          pipx install cmake
+          pipx install ninja
+          cmake --version
+          ninja --version
+
+      - name: Build wheels
+        uses: joerick/cibuildwheel@v1.11.0
+        env:
+          CIBW_BUILD: "${{ matrix.build }}"
+          CIBW_ENVIRONMENT_MACOS: "MACOSX_DEPLOYMENT_TARGET=10.10"
```

If I understand the cibuildwheel docs correctly, we could add those archs to get aarch64 and ppc64le builds via qemu emulation on GitHub CI:
```suggestion
          CIBW_ARCHS_LINUX: auto aarch64 ppc64le
          CIBW_ENVIRONMENT_MACOS: "MACOSX_DEPLOYMENT_TARGET=10.10"
```
Alternatively, if you have Travis-CI credits, we can run ppc64le on Travis-CI's native runners (looks like you already did this part :)).

---

> Alternatively, if you have Travis-CI credits, we can run ppc64le on Travis-CI's native runners (looks like you already did this part :)).

The GHA emulated build would work (but I wouldn't do it only by adding `CIBW_ARCHS_LINUX`, I would also update the matrix to get some parallelism because it would be really slow...). I would expect the build to take around 2 hours where it's less than 20 minutes on Travis-CI.

It seems that it's free again on Travis-CI even though I couldn't find an announcement by either Travis-CI or IBM. ARM released [one](https://community.arm.com/developer/tools-software/tools/b/tools-software-ides-blog/posts/travis-ci-partner-queue-solution-for-open-source-projects) and Travis-CI updated their [doc](https://docs.travis-ci.com/user/billing-overview/#usage---credits) to mention the "Partner Queue" with "0 Credits per
started build minute".

---

That's awesome, thanks for the details!